octoprint_mrbeam/__init__.py

```diff
@@ -738,6 +739,15 @@ def calibration_tool_mode(self):
         self._fixEmptyUserManager()
         return ret
 
+    @property
+    def user_checks_for_updates(self):
```

Comment:
suggest the following refactor:
`user_checks_for_updates` --> `explicit_Update_Check`

Reply:
I can do this